docs/serving/installing-cert-manager.md

```diff
@@ -0,0 +1,121 @@
+[Configuring HTTPS with TLS certificates](./using-an-ssl-cert.md).
+
+You can use cert-manager to either manually obtain certificates, or to enable Knative for 
+automatic certificate provisioning. If you want to enable Knative to automatically obtain
+new TLS certificates and renew existing ones, you must also configure the `config-certmanager` 
+ConfigMap. Complete instructions about automatic certificate provisioning are provided in 
+[Enabling automatic TLS cert provisioning](./using-auto-tls.md).
+
+Regardless of if your want to manually obtain certificates, or configure Knative for automatic
+provisioning, you can use the follwoing steps to install cert-manager.
+
+## Before you begin
+
+You must meet the following requirements to install cert-manager for Knative:
+
+- Knative Serving must be installed. For details about installing the Serving
+  component, see the [Knative installation guides](../install/).
+- You must configure your Knative cluster to use a
+  [custom domain](./using-a-custom-domain.md).
+- Knative currently supports cert-manager version `0.6.1` or higher.
+
+## Downloading and installing cert-manager
+
+Use the following steps to download, install, and configure cert-manager for your Knative cluster
+environment:
+
+1. Run the following commands to download and extract the `cert-manager` installation package:
+
+    ```shell
+    CERT_MANAGER_VERSION=0.6.1
+    DOWNLOAD_URL=https://github.com/jetstack/cert-manager/archive/v${CERT_MANAGER_VERSION}.tar.gz
+
+    wget $DOWNLOAD_URL
+    tar xzf v${CERT_MANAGER_VERSION}.tar.gz
+
+    cd cert-manager-${CERT_MANAGER_VERSION} 
+    ```
+
+2. Depending on how you want to use cert-manager, choose one of the following
+   options for installing cert-manager:
+
+    - **Manually obtain and renew certificates**: Run the following command to 
+      install the cert-manager CRDs for Knative so that you can use the cert-manager tool for 
+      manually obtaining TLS certificates and adding them to your cluster.
+
+       ```shell
+       kubectl apply -f deploy/manifests/00-crds.yaml
+       ```
+
+    - **Enable Knative to automatically obtain and renew TLS certificates**: 
+    
+      1. Run one of the following commands to enable Knative to automatically obtain 
+         new TLS certificates and renew existing ones:
+      
+         - For Knative clusters running Kubernetes version 1.13 or above:
+
+           ```shell
+           # If you are running cluster in 1.13 or above
+           kubectl apply -f deploy/manifests/cert-manager.yaml 
+           ```
+
+         - For Knative clusters running Kubernetes version 1.12 or below:
+
+           ```shell
+           # If you are running cluster in 1.12 or below, you will need to add the --validate=false flag
+           kubectl apply -f deploy/manifests/cert-manager.yaml --validate=false
+           ```
+          
+         <!-- TODO(zhiminx) add the link of networking-certmanager deployment after the code is checked in.-->
```

Comment:
@ZhiminXiang its not clear what this is or how to determine if `networking-certmanager deployment` is installed?

Reply:
As we discussed, this should point to the "using-auto-tls.md" file.